internal/orchestrator/helpers.go

```diff
@@ -73,27 +90,33 @@ func parseAppStatus(containers []container.Summary) []AppStatusInfo {
 		appPath := paths.New(appPath)
 
 		//	running: all running
-		if !slices.ContainsFunc(s, func(v Status) bool { return v != StatusRunning }) {
+		if !slices.ContainsFunc(s, func(v containerStateInfo) bool { return v.State != StatusRunning }) {
 			appendResult(appPath, StatusRunning)
 			continue
 		}
 		//	stopped: all stopped
-		if !slices.ContainsFunc(s, func(v Status) bool { return v != StatusStopped }) {
+		if !slices.ContainsFunc(s, func(v containerStateInfo) bool { return v.State != StatusStopped }) {
 			appendResult(appPath, StatusStopped)
 			continue
 		}
 
 		// ...else we have multiple different status we calculate the status
 		// among the possible left: {failed, stopping, starting}
-		if slices.ContainsFunc(s, func(v Status) bool { return v == StatusFailed }) {
+		if slices.ContainsFunc(s, func(v containerStateInfo) bool { return v.State == StatusFailed }) {
+			appendResult(appPath, StatusFailed)
+			continue
+		}
+		if slices.ContainsFunc(s, func(v containerStateInfo) bool {
+			return v.State == StatusStopped && strings.Contains(v.StatusMessage, "Exited (0)")
```

Comment:
We should account for all exit codes that aren't signals, so everything that is less than 128

If you write an app with Python-like

```python
exit(1)
```

You say that this isn't a fail, which in fact it is, because the app exits right after be started.